### ISSUE TITLE
Charts are unavailable when multiple charts are on one page

### ISSUE THREAD

**Issue by Asyvix** [Live-Charts/LiveCharts2#957]
Multiple charts cannot be placed on a single page due to browser WebGL limitations. If create more than about 8(almost) charts, they will no longer be created or disable previously created charts. I think it need a function that can group multiple charts in one context.

**To Reproduce**
create blazor webassembly project and add 10 chart controls.

**Expected behavior**
work fine

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 11
 - Browser [e.g. chrome, safari] Microsft Edge(Chromium), Chrome

**Smartphone (please complete the following information):**

**Additional context**

**Comment by beto-rodriguez** [comment]
This is now "fixed"

Since the next version of the library we will be able to turn off WebGL by calling `LiveCharts.UseGPU = false;`

**Comment by GamiDroid** [comment]
@beto-rodriguez I use LiveCharts2 in my Blazor WASM application, I just upgraded the package to 2.0.0-rc6.1. 
Now the setting `LiveCharts.UseGPU` is not available anymore. I noticed the setting `LiveCharts.RenderingSettings.UseGPU = false;` is now possible.

But now the bug returned. When I have multiple charts in the screen. The first WebGL context is removed. Is there another way to disable WebGL in this version?

I have the following warning in the Browser console: "WARNING: Too many active WebGL contexts. Oldest context will be lost.".

**Comment by beto-rodriguez** [comment]
@GamiDroid You are right, I removed that option. I re-opoened so we can fix this again, sorry about that!

**Comment by GamiDroid** [comment]
dear @beto-rodriguez, Can you give any updates?

**Comment by beto-rodriguez** [comment]
Fix coming in #2210, sorry for the delay @GamiDroid.

Root cause: `MotionCanvas.razor` in `LiveChartsCore.SkiaSharpView.Blazor` was hardcoded to `SKGLView`, so the `LiveCharts.RenderingSettings.UseGPU = false` setting silently had no effect on Blazor — every chart kept its own WebGL context regardless. Browsers cap the number of live WebGL contexts (Chromium drops the oldest at ~8), which is exactly the symptom you reported with 10 charts on a page.

In #2210 the Blazor `MotionCanvas` actually honours `UseGPU`:
- `UseGPU = true` (the default — preserves the existing look) → `SKGLView` / WebGL.
- `UseGPU = false` → `SKCanvasView` / raster, no WebGL contexts at all.

After the PR lands, your scenario should work by setting it once in `App.razor`'s `OnInitialized`:

```csharp
LiveCharts.Configure(c => c
    .HasRenderingSettings(s => s.UseGPU = false));
```

Quality note: raster output is visibly lower-quality than the WebGL render, especially for thin lines and antialiased edges. So this is an opt-in based on your scenario (many charts on a page, sandboxed iframe, etc.), not the default.

**Comment by beto-rodriguez** [comment]
> AI-drafted note posted on Beto's behalf.

Fixed in **LiveCharts 2.0.3**, now available on NuGet (see PR #2210).
